_data/languages.yml

```diff
@@ -2,7 +2,7 @@
   tag: "en"
   enabled: No
   accept_languages: ["en", "en-us", "en-au", "en-nz", "en-za", "en-bz", "en-tt"]
-  site_name: "Bisq - The decentralized Bitcoin exchange"
+  site_name: "Bisq - A decentralized Bitcoin exchange network"
   site_desc: "Bisq is an open-source desktop application that allows you to buy and sell bitcoins in exchange for national currencies, or alternative cryptocurrencies."
```

Comment:
I realize that this text isn't part of the proposed pull request, but I suggest we change it to match the wording in the introduction of https://docs.bisq.network/exchange/whitepaper.html:

> Bisq is an open source peer-to-peer application that allows anyone to buy and sell Bitcoin in exchange to national currencies or alternative crypto currencies.

and actually, I'd suggest we tweak _that_ wording as follows:

Bisq is an open source, peer-to-peer application that allows anyone to buy and sell ~~B~~bitcoin in exchange ~~to~~for national currencies ~~or~~and other ~~crypto currencies~~cryptocurrencies.

such that it reads like this:

> Bisq is an open source, peer-to-peer application that allows anyone to buy and sell bitcoin in exchange for national currencies and other cryptocurrencies.

A quick search will show that minor variations of the above wording have been used in many docs and articles around the web.

Reply:
Agreed. I've submitted [another PR for this](https://github.com/bisq-network/bisq-website/pull/303) since others have already reviewed this one.